cmd/plakar/subcommands/backup/stdio.go

```diff
@@ -18,17 +17,17 @@ func eventsProcessorStdio(ctx *context.Context, quiet bool) chan struct{} {
 		for event := range ctx.Events().Listen() {
 			switch event := event.(type) {
 			case events.PathError:
-				logger.Warn("%x: KO %s %s: %s", event.SnapshotID[:4], crossMark, event.Pathname, event.Message)
+				ctx.GetLogger().Warn("%x: KO %s %s: %s", event.SnapshotID[:4], crossMark, event.Pathname, event.Message)
 			case events.DirectoryOK:
 				if !quiet {
-					logger.Info("%x: OK %s %s", event.SnapshotID[:4], checkMark, event.Pathname)
+					ctx.GetLogger().Info("%x: OK %s %s", event.SnapshotID[:4], checkMark, event.Pathname)
 				}
 			case events.FileOK:
 				if !quiet {
-					logger.Info("%x: OK %s %s", event.SnapshotID[:4], checkMark, event.Pathname)
+					ctx.GetLogger().Info("%x: OK %s %s", event.SnapshotID[:4], checkMark, event.Pathname)
 				}
 			default:
-				//logger.Warn("unknown event: %T", event)
+				//logging.Warn("unknown event: %T", event)
```

Comment:
it's commented so i don't think it'll matter, but it should be `ctx.GetLogger().Warn` here too :P

Reply:
fixed